app/main/views/service_updates.py

```diff
@@ -50,37 +52,60 @@ def service_status_update_audits(day=None, page=1):
     )
 
 
-@main.route('/service-updates', methods=['GET'])
+_service_update_audits_event_limit = 1000
```

Comment:
Eh, I don't think it implies it's dickable-with if it doesn't have a leading underscore. To me, global mutables are lowercase, global constants are uppercase. I don't mind if you have a leading underscore either way.

Google Python style guide (which I generally think is pretty good: http://google.github.io/styleguide/pyguide.html#Global_variables) agrees, although I accept that's not the source of truth for our repositories.